### PR TITLE
Fix on Content-Length header for HTTP2->HTTP1 to make POSTs work

### PR DESCRIPTION
**Pull request by dstockton**
https://github.com/cloudflare/cloudflared/issues/30

### CODE REVIEW

**Comment by dstockton** [comment]
@sssilver I've re-raised https://github.com/cloudflare/cloudflared/issues/30 / https://github.com/cloudflare/cloudflared/pull/28 here; can you review please?

File: origin/tunnel.go [hunk 1]
```diff
@@ -431,6 +431,12 @@ func H2RequestHeadersToH1Request(h2 []h2mux.Header, h1 *http.Request) error {
 				return fmt.Errorf("invalid path")
 			}
 			h1.URL = resolved
+		case "content-length":
+			l, err := strconv.ParseInt(header.Value, 10, 64)
```
Comment (sssilver) [inline origin/tunnel.go]: Although the scope is tiny, let's call this `contentLength`.

**Comment by dstockton** [comment]
@sssilver - thanks for the review. I've made the change with the commit message linking to the issue as requested.
Many thanks for your time,
David

**Comment by sssilver** [comment]
Thanks @dstockton, it looks like you still have 2 commits in your log :) Please squash them into a single commit and force push.

**Comment by dstockton** [comment]
@sssilver Squashed - Thanks for your patience!!!

**Comment by sssilver** [comment]
Many thanks, @dstockton!

**Pull request merged**